ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ip_nat.schema.yml

```diff
@@ -38,26 +38,32 @@ keys:
           keys:
             name:
               type: str
+            type:
+              type: str
+              valid_values: ["ip-port", "port-only"]
+              default: "ip-port"
             prefix_length:
               type: int
-              required: true
               convert_types:
                 - str
               min: 16
               max: 32
+              description: It is only used and required when `type` is `ip-port`.
             ranges:
               type: list
               items:
                 type: dict
                 keys:
                   first_ip:
                     type: str
-                    required: true
-                    description: IPv4 address
+                    description: |-
+                      IPv4 address.
+                      Required when `type` is `ip-port` and ignored otherwise.
                   last_ip:
                     type: str
-                    required: true
-                    description: IPv4 address
+                    description: |-
+                      IPv4 address.
+                      Required when `type` is `ip-port` and ignored otherwise.
```

Comment:
Add below:
Start and End Ip addresses should be in same subnet

Reply:
Done